src/chainparams.cpp

```diff
@@ -796,7 +796,7 @@ class CRegTestParams : public CChainParams {
         consensus.DIP0024QuorumsHeight = 1; // Always have dip0024 quorums unless overridden
         consensus.V19Height = 1; // Always active unless overriden
         consensus.V20Height = consensus.DIP0003Height; // Active not earlier than dip0003. Functional tests (DashTestFramework) uses height 100 (same as coinbase maturity)
-        consensus.MN_RRHeight = 900;
+        consensus.MN_RRHeight = 1;
```

Comment:
mn_rr should not activate prior to v20 so maybe `= consensus.V20Height;`?

Reply:
I tried 78bb23a and it doesn't work (rpc_masternodes.py fails because mn_rr doesn't follow height).

mn_rr should be adjusted on regtest everytime when v20 activation is adjusted. It adds unneded complexity

Reply:
pls consider 6785d564c73df942172b91ba73c845da4f5d8664